docker/base_images/base_image.Dockerfile.jinja

```diff
@@ -34,7 +34,6 @@ RUN apt update && \
     add-apt-repository -y ppa:deadsnakes/ppa && \
     apt update -y && \
     apt install -y python{{python_version}} && \
-    apt install -y python{{python_version}}-distutils && \
```

Comment:
why did we need to do this? could we leave this out of the change here?

Reply:
ie: do we still need this for older pythons?

Reply:
the packages that replace distutils (shutil, packaging) are available in all available versions of truss (3.8 through 3.12), so I don't think we would need to keep the distuitls install around